doc/devel/contribute.rst

```diff
@@ -158,6 +158,25 @@ please reach out on the :ref:`contributor_incubator`
 
 .. _`open an issue`: https://github.com/matplotlib/matplotlib/issues/new?assignees=&labels=Documentation&projects=&template=documentation.yml&title=%5BDoc%5D%3A+
 
+.. _contribute_triage:
+
+Triage
+------
+We appreciate help keeping our `issue tracker <https://github.com/matplotlib/matplotlib/issues>`_
+organized because the issue tracker is our centralized location for feature requests,
+bug reports, tracking major projects, and discussing priorities.
+
+Some examples of what we mean by triage are:
+
+* labeling issues and pull requests
+* verifying bug reports
+* debugging and resolving issues
+* linking to related issues, discussion, and external work
+
+Helping out with triage is discussed in detail in :ref:`bug_triaging`.
```

Comment:
With my non-native English, I feel "helping out" is not that welcoming. I read it as "We have a deficiency. Can you temporarily take over a task that we currently cannot fulfill".

I suggest either

```suggestion
Helping with triage is discussed in detail in :ref:`bug_triaging`.
```
or
```suggestion
These tasks are discussed in detail in :ref:`bug_triaging`.
```
But you know the language better, so please decide whether to keep as is or change.